getting-started/templates/systemlink-secrets.yaml

```diff
@@ -251,6 +251,37 @@ dataframeservice:
           accessKey: *minioUser # <ATTENTION> - change if not using MinIO
           secret: *minioPassword # <ATTENTION> - change if not using MinIO
 
+## Secret configuration for feedservice
+##
+feedservice:
+  secrets:
+    ## Access key information for S3/MinIO access.
+    ## If using MinIO, use the user and password from the MinIO configuration above here. Otherwise, S3 credentials will need to be provided.
+    ##
+    s3:
+      ## Access key ID for the S3 instance. Override if not using MinIO.
+      ##
+      accessKeyId: *minioUser
+      ## Access key for the S3 instance. Override if not using MinIO.
+      ##
+      accessKey: *minioPassword
+      ## Default region for the S3 instance. Override if not using MinIO.
+      ##
+      defaultRegion: "us-east-1"
```

Comment:
Should this also be a referenced variable rather than a hardcoded string. Like the user and password?

Reply:
Used yaml anchor to reuse the value.